streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -98,8 +97,9 @@
 
     private StreamPartitionAssignor partitionAssignor = null;
 
-    private long lastClean;
-    private long lastCommit;
+    private long currTimeMs;
```

Comment:
ok - i still think `timerStartedMs` is a better name. i can't think of anything else